ProcessRGB.cpp

```diff
@@ -1902,6 +2117,75 @@ static etcpak_force_inline int GetMulSel( int sel )
         return 5;
     }
 }
+
+#endif
+
+#ifdef __ARM_NEON
+
+static constexpr etcpak_force_inline int GetMulSel(int sel)
+{
+    return (sel < 1) ? 0 : (sel < 4) ? 1 : (sel < 5) ? 2 : (sel < 8) ? 3 : (sel < 14) ? 4 : 5;
+}
+
+static constexpr int ClampConstant(int x, int min, int max)
+{
+    return x < min ? min : x > max ? max : x;
```

Comment:
Adapting code from `clampu8()` (Math.hpp:74) might result in faster code.

Reply:
(Well, probably not, as it's compile time constant...)

Reply:
This is just used as a workaround to make it compile after I removed use of `if constexpr` (which requires a compiler that supports C++17).